build-win.ps1

```diff
@@ -76,7 +76,7 @@ foreach ($arch in $archs) {
         echo ""
         continue
       }
-      if ($binType -eq "lib" -and ($actualArch -eq "ARM" -or $actualArch -eq "ARM64")) {
```

Comment:
If this is the only essential change necessary, can the other changes in .sln file all be safely reverted?

Reply:
No - the other solution changes for arm64 are also needed. The only optional one was to remove the references to SampleCppUTC.